guide/creating-your-bot/common-questions.md

```diff
@@ -109,6 +109,17 @@ user.send('<content>');
 
 <p class="tip">If you want to learn more about this syntax or want to learn about reaction collectors as well, check out [this dedicated guide page for collectors](/creating-your-bot/collectors)!</p>
 
+### How do I react to the message my bot sent?
+
+```js
+<message>.channel.send('My message to react to.').then(sentMessage => {
+	sentMessage.react('👍');
+	sentMessage.react(<client>.emojis.get('myCustomEmojiIdHere'));
+})
+```
+
+<p class="tip">If you want to learn more about reactions, check ou [this dedicated guide on reactions](/additional-info/reactions)!</p>
```

Comment:
Typo at "check ou", and missing a semicolon at the end of the snippet. Other than that, looks good to me. 👍